tests/mc_simulator/test_mc_simulator.py

```diff
@@ -73,6 +79,10 @@
     bytes(0),
 )
 
+DEFAULT_SIMULATOR_STATUS_CODE = SERIAL_COMM_BOOT_UP_CODE.to_bytes(
+    SERIAL_COMM_STATUS_CODE_LENGTH_BYTES, byteorder="little"
```

Comment:
this seems to come up frequently. for future could consider creating
`def convert_status_code_to_bytes(code:int)->bytes`